pkg/server/server.go

```diff
@@ -83,6 +84,7 @@ const (
 	sha256Prefix                          = "sha256~"
 	tokenizerPageTemplateName             = "tokener.html"
 	updatesEndpoint                       = "/api/check-updates"
+	crdSchemaEndpoint                     = "/api/console/crd-schema"
```

Comment:
nitpick: I'd probably make this `/api/crd-columns`, which matches more closely to the other paths we have. Since we're only returning columns and not the JSON schema, `crd-columns` is probably more accurate.